index.html

```diff
@@ -12,34 +12,47 @@
     <div class="center-wrapper three-column-grid__expand-two">
         <aside class="left-menu asides-padding">
             <ul>
-                <img class="margin-bottom-md" src="/icons/twitter.png" alt="Twitter icon.">
```

Comment:
I never thought to add the img-tag on line 15, between the ul-tag and li-tag to keep it in the ul container - better for alignment and styling?

Reply:
The idea behind keeping the logo image inside the ul at first was to treat it like a "home" button linking to the main page, so it would be part of the navigation like the others. But since there is an actual "home" button already I ended up not implementing that on the logo and left it like this for simplicity when aligning elements.